modules/visual_script/visual_script_editor.cpp

```diff
@@ -3515,6 +3515,7 @@ void VisualScriptEditor::_selected_connect_node(const String &p_text, const Stri
 	}
 
 	Ref<VisualScriptNode> vnode;
+	Ref<VisualScriptPropertySet> scriptPropSet;
```

Comment:
We use `snake_case` for identifiers, so this should be `script_prop_set`.

Reply:
Woops, my bad. Fixed this :+1:  @akien-mga